docs/evm-sidechain/xrpl-evm-sidechain-faq.md

```diff
@@ -0,0 +1,104 @@
+---
+html: xrpl-evm-sidechain-faq.html
+parent: intro-to-evm-sidechain.html
+blurb: Commonly asked legal questions to consider when running an EVM compatible sidechain.
+labels:
+  - Blockchain
+---
+# XRPL EVM Sidechain FAQ
+
+### Why are Ripple and PeerSyst building an EVM sidechain?
+
+The XRPL natively lacks the full programmability option to add custom business logic. The only programmabilIty option XRPL has today is a “smart transactor” but it's very difficult to write one and any significant change that affects existing protocol needs to go through the amendment process to be approved by the XRPL community. The Hooks feature developed by XRPL Labs promises to bring native programmability to the XRPL. EVM compatible blockchains have “smart contract” functionality that enables developers to build and deploy new applications without a network vote.
+
+EVM compatibility on different blockchains:
+
+| Blockchain Ecosystem | EVM Compatibility Solution/Project |
+|----------------------|------------------------------------|
+| Ethereum             | Native                             |
+| Solana               | Neon                               |
+| Polkadot             | Moonbeam                           |
+| Cosmos               | Evmos                              |
+| Polygon              | zkEVM                              |
+| BNB Chain            | BNB Smart Chain                    |
+| Avalanche            | Avalance C-chain                   |
+| **XRPL**             | **EVM Sidechain**                  |
+
+An EVM sidechain will help the XRPL ecosystem attract EVM developers who can deploy their existing EVM applications on the EVM sidechain with minor modifications.
+
+
+### What are the main benefits of the EVM sidechain?
+
+The main benefits are:
+
+- Faster time to launch by deploying applications as smart contracts.
+- No amendment approval needed to launch applications.
+- Make minor changes to existing smart contracts on other blockchains and launch them on the EVM sidechain.
+- Grow the XRPL ecosystem by adding EVM ecosystem projects and developers.
+
+
+### What are the main risks with the EVM sidechain?
+
+With the XRPL, developers use existing primitives such as payments, DEX, escrow etc. which are secure and have a great track record. With EVM applications, the security is the responsibility of the smart contract developer. Smart contract bugs can get exploited and they have led to a loss of funds.
+
+
+### What is the main use case for the EVM sidechain?
+
+From an XRPL community perspective, the EVM sidechain enables EVM developers to launch their existing applications in the XRPL ecosystem. Developers can now write smart contracts and build applications, and not be constrained by the capabilities of XRPL smart transactors. XRPL developers can launch applications that previously required XRPL protocol change and an amendment proposal which is complex, resource intensive, and risky.
```

Comment:
Not sure we want to diss smart transactors. We can just say they're getting enhanced behavior. I'd check with PM on this.